core/migrate/data.go

```diff
@@ -117,13 +117,16 @@ var migrations = []migration{
 	{Name: "2017-01-25.0.account.cp-expiry.sql", SQL: `
 		ALTER TABLE account_control_programs ADD COLUMN expires_at timestamp with time zone;
 	`},
-	{Name: "2017-01-30.0.txdb.snapshots-timestamp.sql", SQL: `
+	{Name: "2017-01-30.1.txdb.snapshots-timestamp.sql", SQL: `
 		ALTER TABLE snapshots ADD COLUMN created_at timestamp without time zone DEFAULT now();
 	`},
-	{Name: "2017-01-30.1.core.add-block-hsm-config.sql", SQL: `
+	{Name: "2017-01-30.2.core.add-block-hsm-config.sql", SQL: `
```

Comment:
Blerp. 

I think I'm going to remove the schema changes from this PR altogether and open a new one with the very simple db change we're actually interested in below.